### ISSUE TITLE
Invalid warning 'Sentence must end with a period'

### ISSUE THREAD

**Issue by wdanilo** [gajus/eslint-plugin-jsdoc#942]
Hi, I've got such a file:

```ts
/** @file Easing is a method of distorting time to control apparent motion in animation. This module
 * defines a set of easing functions. To learn more about them and see graphs of how they behave,
 * see: https://github.com/d3/d3-ease. */

// =================
// === Constants ===
// =================

/** Two power minus ten times t scaled to [0,1]. */
export function tpmt(x: number) {
    return (Math.pow(2, -10 * x) - 0.0009765625) * 1.0009775171065494
}

...
```

And I've got a warning:

```
2:1  warning  Sentence must end with a period                            jsdoc/require-description-complete-sentence
```

Which seems incorrect, as all the sentences end with a dot.

**Comment by brettz9** [comment]
The bug is apparently connected to the fact that the final `*/` is on the same line.

**Comment by wdanilo** [comment]
@brettz9 oh, thanks so much for fixing it so fast!!

**Comment by brettz9** [comment]
Sure. Hopefully we can get a release out for it soon, as the automated release system was having some issues.
